ci-operator/step-registry/aro-hcp/provision/delete-all-expired-resource-groups/aro-hcp-provision-delete-all-expired-resource-groups-commands.sh

```diff
@@ -4,15 +4,12 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-set -x # Turn on command tracing
-
 unset GOFLAGS
 make -C test/
 
-# read the secrets and login as the user
-export TEST_USER_CLIENT_ID; TEST_USER_CLIENT_ID=$(cat /var/run/hcp-integration-credentials/client-id)
-export TEST_USER_CLIENT_SECRET; TEST_USER_CLIENT_SECRET=$(cat /var/run/hcp-integration-credentials/client-secret)
-export TEST_USER_TENANT_ID; TEST_USER_TENANT_ID=$(cat /var/run/hcp-integration-credentials/tenant)
-az login --service-principal -u "${TEST_USER_CLIENT_ID}" -p "${TEST_USER_CLIENT_SECRET}" --tenant "${TEST_USER_TENANT_ID}"
+# use login script from the aro-hcp-provision-azure-login step
+/bin/bash "${SHARED_DIR}/az-login.sh"
+
+set -x # Turn on command tracing
 
-./test/aro-hcp-tests delete-expired-resource-groups
\ No newline at end of file
+CUSTOMER_SUBSCRIPTION="${SUBSCRIPTION}" ./test/aro-hcp-tests delete-expired-resource-groups
```

Comment:
If there's already a CLI to do half of this, just teach it how to clean up HCP clusters and RGs, and you're down to two steps that need to log into Azure - the test runner and this cleanup step. (My other PR deletes the rest of these, they are unused). Then the horror of logging in manually in both is very small.